app/main.py

```diff
@@ -1,3 +1,20 @@
 class OnlineCourse:
-    # write your code here
-    pass
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        if days % 7 != 0:
+            return (days // 7) + 1
+        return days / 7
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        return OnlineCourse(
```

Comment:
```suggestion
        return cls(
```
It's better to use the `cls` parameter to access  in the classmethods

Reply:
The reasons will be discussed in the following topics